handlers/meshery_application_handler.go

```diff
@@ -129,8 +129,8 @@ func (h *Handler) handleApplicationPOST(
 			if err != nil {
 				obj := "convert"
 				h.log.Error(ErrApplicationFailure(err, obj))
-				http.Error(rw, ErrApplicationFailure(err, obj).Error(), http.StatusInternalServerError) // sending a 500 when we cannot convert the file into kuberentes manifest
-				return
+				// http.Error(rw, ErrApplicationFailure(err, obj).Error(), http.StatusInternalServerError) // sending a 500 when we cannot convert the file into kuberentes manifest
```

Comment:
@sudo-NithishKarthik  How will this make a fix? I mean here instead of throwing error to client, we will be just putting an empty string as Application file.